playwright.config.js

```diff
@@ -2,16 +2,27 @@ import { defineConfig } from '@playwright/test';
 
 export default defineConfig({
   fullyParallel: true,
-  reporter: process.env.CI ? 'blob' : 'html',
+  outputDir: './dist/playwright',
+  reporter: process.env.CI ? 'blob' : 'line',
+
+  // For whatever reason, Playwright can't find snapshots that exist
+  // in an subdirectory of `outputDir`. So they're at the top level.
   snapshotPathTemplate: './dist/baseline-screenshots/{arg}{ext}',
+
   testDir: './src/',
   testMatch: /.*\.test\.visuals\.ts/,
+
+  // 30 seconds is the default. Each test should only take a couple seconds.
+  // Something has gone wrong if one takes longer. So we fail fast to give
+  // developers feedback.
+  timeout: 5000,
+
   use: {
     baseURL: 'http://localhost:6006',
   },
   webServer: {
     command: 'pnpm start:development:storybook',
-    reuseExistingServer: !process.env.CI,
+    reuseExistingServer: true,
```

Comment:
Playwright would fail if `pnpm test:development:playwright` is run but not `pnpm start`. Now Playwright will start Storybook locally if it's not already running.

Reply:
Nice improvement!